zulip/integrations/google/doc.md

```diff
@@ -1,72 +1,172 @@
+
+1.  Optionally, to configure the bot, add a **google-calendar** section to
+    the `.zuliprc` file, like below:
+
+    ```
+    [google-calendar]
+    calendar=primary
+    interval=3600
+    tokens_path=/home/username/zulip/integrations/google-oauth-tokens.json
+    client_secret_path=/home/username/zulip/integrations/client-secret-credentials.json
+    channel=core-team
+    topic=scheduled events
+    noauth_local_webserver=
+    format_message=The event [{title}]({calendar_link}), at {location}, is from {start} to {end}.\n> {description}\n\n[Join call]({google_meet_link}).
+    ```
+
+    See [configuration options](#configuration-options) for the list of
+    available options.
+
+{end_tabs}
+
+### Setup Google OAuth Client ID
+
+{start_tabs}
+
+!!! tip ""
+
+    A free Google account is sufficient for this integration, a Google
+    service account is not required.
+
+1.  In the Google Cloud console, go to
+    **Menu > Google Auth platform > [Clients][clients-menu]**. Click
+    **Create Client**.
+
+[clients-menu]: https://console.cloud.google.com/auth/clients
+
+1.  Select **Application type > Desktop app**. Set **Name** to a name of
+    your choice, such as `Zulip`, and click **Create**. Save the downloaded
+    JSON file as `client_secret.json` in your `~/` directory.
+
+1.  From your Google Cloud dashboard, go to **APIs & Services** and select
+    **OAuth consent screen**. Under **Test users**, click **+ Add Users**.
+    Enter the email address corresponding to your Google Calendar, and click
+    **Save**.
+
+{end_tabs}
+
+### Run the integration script
+
+{start_tabs}
+
+1.  Install the requirements for the integration script with:
+
+    `python {{ integration_path }} --provision`
+
+1.  Run the integration script with:
+
+    `python {{ integration_path }}`
+
+    Authorize access in the browser window that opens, to allow the Zulip
+    bot to view your Calendar. If you've turned on the
+    `noauth_local_webserver` option, follow the terminal prompts instead,
+    and paste the resulting authorization code.
+
+1.  Optionally, pass [commandline arguments](#configuration-options) to
+    configure the integration. The commandline arguments override the
+    corresponding settings in the `.zuliprc` file.
+
+    ```
+    python {{ integration_path }} \
+    --calendar primary \
+    --interval 3600 \
+    --client-secret-file /home/username/zulip/integrations/client-secret-credentials.json \
+    --tokens-file /home/username/zulip/integrations/google-oauth-tokens.json \
+    --channel core-team \
+    --topic scheduled-events \
+    --format-message "The event [{title}]({calendar_link}), at {location}, is from {start} to {end}.\n> {description}\n\n[Join call]({google_meet_link})."
+    ```
+
+    See [configuration options](#configuration-options) for the list of
+    available options.
+
+1.  You will get reminders as long as the terminal session with the bot
```

Comment:
Instead of "reminders", I'd say "notifications" since that's what we called them in the description at the top.